libraries/Microsoft.Bot.Builder/TelemetryLoggerMiddleware.cs

```diff
@@ -46,6 +59,15 @@ public TelemetryLoggerMiddleware(IBotTelemetryClient telemetryClient, bool logPe
         [JsonIgnore]
         public IBotTelemetryClient TelemetryClient { get; }
 
+        /// <summary>
+        /// Gets The telemetry client to send telemetry events to.
+        /// </summary>
+        /// <value>
+        /// The <see cref="IBotTelemetryClient"/> this middleware uses to log events.
```

Comment:
We're saying that `IBotTelemetryClient` is obsolete now, so this comment probably shouldn't refer to it?

Reply:
I was marking it as obselete to indicate that a user that they should update their config but leaving in place so that existing solutions still work - based on our discussion, I agree that we should remove the obselete flag though until R9. Give us change to update docs / samples / advice over that period and get our updated telemetry story out there.